.jshintrc

```diff
@@ -0,0 +1,63 @@
+{
+	"bitwise": false,
+	"camelcase": false,
+	"curly": true,
+	"eqeqeq": false,
+	"forin": true,
+	"freeze": true,
+	"immed": false,
+	"latedef": true,
+	"newcap": true,
+	"noempty": true,
+	"nonew": true,
+	"noarg": true,
+	"plusplus": false,
+	"quotmark": "single",
+	"undef": true,
+	"maxparams": 5,
+	"maxdepth": 5,
+	"maxstatements": 30,
+	"maxlen": 100,
+	"nonbsp": true,
+	"esversion": 9,
+	"browser": false,
+	"regexp": true,
+	"strict": true,
+	"trailing": false,
+	"sub": true,
+	"smarttabs": true,
+	"lastsemic": false,
+	"globals": {
+		"document": true,
+		"console": true,
+		"require": true,
+		"define": true,
+		"exports": true,
+		"App": true,
+		"Loader": true,
+		"_": true,
+		"google": true,
+		"module": true,
+		"THREE": true,
+		"TWEEN": true,
+		"window": true,
+		"clearInterval": true,
+		"clearTimeout": true,
+		"FileReader": true,
+		"FormData": true,
+		"history": true,
+		"localStorage": true,
+		"location": true,
+		"navigator": true,
+		"screen": true,
+		"sessionStorage": true,
+		"setInterval": true,
+		"setTimeout": true,
+		"Storage": true,
+		"URL": true,
+		"XMLHttpRequest": true,
+		"IntersectionObserver": true,
+		"MutationObserver": true,
+		"ResizeObserver": true
```

Comment:
I would keep native APIs behind the window scope. It's faster to read and understand that's a global scope accessible ressource.

Reply:
(that's why `browser` is at false actually) https://jshint.com/docs/options/#browser